### DISCUSSION TITLE
Better network monitor filtering

### DISCUSSION THREAD

**Discussion by jaril** [RecordReplay/devtools#7349] [Ideas]
<img width="621" alt="image" src="https://user-images.githubusercontent.com/15959269/177838097-752f0df7-2091-4f44-857f-7becd9884c96.png">

tl;dr filter network requests by domain (and other columns)

I have lots of network requests to a variety of vendors as well as different domains for my backend.  It would be really nice to be able to filter the network tab by domain in the same way that Chrome provides. For example domain:*.test would only show requests from domains like api.mydomain.test and app.mydomain.test and not www.vendor.com.

**Comment by hovissimo** [comment]
If you want your network filter to be BETTER than Chrome's, please also remember recent filters.  It's super annoying to have to re-type my different filters in Chrome!

**Comment by jasonLaster** [comment]
Good call. Recent / Saved filters could be a big deal
